iphone/Classes/TiUIWebView.m

```diff
@@ -518,6 +519,15 @@ - (void)setUrl_:(id)args
 	}
 }
 
+-(void)setBlacklistedURLs_:(id)args
+{
+    ENSURE_TYPE(args, NSArray);
+    if (blacklistedURLs) {
+        RELEASE_TO_NIL(blacklistedURLs);
+    }
+    blacklistedURLs = [[NSArray alloc] initWithArray:args];
```

Comment:
As this is allocated here and is getting release when dealloc called or new blacklisted urls are getting set. So I think it is right.